server/routes/comments.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+
+const router = express.Router();
+const replies = require('./replies');
+
+router.get('/', (req, res) => {
+  res.status(200).json({ success: true, message: '' });
```

Comment:
just grab this whole line in success response function and call that. That will we useful as then you just have to pass the message and response as the parameter, status and success will already be set.

Reply:
if we do that it will be overkill because the router can be bound to different paths, if we create another module then we will have to pass router, controller, req, res to it and then export it to different modules.